src/components/listSelection/ListSelection.js

```diff
@@ -4,14 +4,25 @@ import './ListSelection.css';
 import line from './line.svg';
 
 class ListSelection extends Component {
+	isLetter(str) {
+		return str.match(/^[a-z]$/i);
```

Comment:
We could make this do more if we wanted, we're really using it to determine if the first character is a letter

```
startsWithLetter(str) {
  return str.match(/^[a-z/);
}

item => this.startsWithLetter(item.name) ? item.name[0].toUpperCase() : '#'
```